### PR TITLE
feat: Mainnet countdown modal

### PR DESCRIPTION
**Pull request by dgca**
## Summary

Adds countdown modal that displays the time left until mainnet launch. Modal is displayed once per session.

- [Preview link](https://website-git-mainnet-countdown-ironfish.vercel.app/)

## Testing Plan

Manual testing

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

**Review by NullSoldier — APPROVED** [review]

**Review by skylarrichard123 — CHANGES_REQUESTED** [review]
Screen Shot 2023-03-20 at 10 46 11 PM
Make top and bottom margin here 32px.

**Comment by skylarrichard123** [comment]
Screen Shot 2023-03-20 at 10 46 49 PM
This is Arial, should be Favorit .

**Comment by skylarrichard123** [comment]
Screen Shot 2023-03-20 at 10 48 17 PM
add 4px border radius to the form field and button

**Comment by skylarrichard123** [comment]
Screen Shot 2023-03-20 at 10 49 21 PM
Left and right button padding to become 32px (currently 22px)
Change button copy to "Get Notified"

**Comment by skylarrichard123** [comment]
Lastly, two things I did not design for but stand out now:
(1) Could we add a hover on the button (can just make it 20% darker, or use or new darker purple hover state from website v2, I'll let you decide).
(2) the white background container at first make me think it was a scroll bar, can we change that to the same pink from the CTA button?

**Comment by skylarrichard123** [comment]
Screen Shot 2023-03-20 at 10 55 19 PM
Can we change the gap btw the field and the button from 16px down to 8px? Ok, fin.

File: src/theme/MainnetCountdown/useTimeLeft.js
Comment (dguenther) [inline src/theme/MainnetCountdown/useTimeLeft.js:8]: Might be a good idea to prevent this from going negative, and possibly do something different when it hits 0, like disable the modal

**Review by dguenther — APPROVED** [review]
Looks good to me once you get Skylar's approval!

File: src/theme/MainnetCountdown/useMailchimpSubscribe.js
Comment (dgca) [reply src/theme/MainnetCountdown/useMailchimpSubscribe.js:5]: It's for reference, this project doesn't use TypeScript :(

File: src/theme/MainnetCountdown/useTimeLeft.js
Comment (dgca) [reply src/theme/MainnetCountdown/useTimeLeft.js:8]: Lol, yeah, good catch.

**Pull request merged**
